### DISCUSSION TITLE
Storing long (multiline?) values.

### DISCUSSION THREAD

**Discussion by vittorio-perera** [KenKundert/nestedtext#46] [Q&A] [answered]
I am not sure if this is supported or even intended but I was wondering if there is a way to split the value of a key on multiple lines without this affecting the actual value. My goal is to store some longer strings (e.g., short paragraph) but keep them readable. To do so I'd like to split the text in multiple lines but still have it as a single one when it's loaded. Thanks for any help!

**Comment by AndydeCleyre** [comment]
I _think_ the current stance on this is to not make this part of the spec, but rather a choice individual projects can make, by replacing the newlines after loading the value. It would be a policy of that project/config/value only, not NestedText itself.

**Reply by KenKundert** [reply, marked as answer]
@AndydeCleyre is correct.  That is our stance.  This is something that can and should be done by the end application on a case-by-case basis.  We have also included an [example](https://nestedtext.org/en/stable/techniques.html#long-lines) of how to do so with a schema.

**Reply by vittorio-perera** [reply]
Great thanks for the quick reply!
